README.md

```diff
@@ -3,7 +3,7 @@
 [![Build Status](https://travis-ci.org/iawia002/annie.svg?branch=master)](https://travis-ci.org/iawia002/annie)
 [![codecov](https://codecov.io/gh/iawia002/annie/branch/master/graph/badge.svg)](https://codecov.io/gh/iawia002/annie)
 
-👾 A fast, simple and clean video downloader built with Go.
+👾 Annie is a fast, simple and clean video downloader built with Go. Annie helps users to download videos off supported websites, such as Youtube. With Annie, downloading videos and playlists has never been so easy!
```

Comment:
It's a slogan, it should be concise, so I think this can't be changed

Reply:
and I don't understand what does `download videos off supported websites` mean, especially **off**

Reply:
'download videos off supported websites' is to download the video from the website. To 'take something off' is a type of idiom. Maybe using 'off' is a bad idea actually. But, if
> A fast, simple and clean video downloader built with Go.

Maybe the introduction could be added afterwards? For example,

👾 Annie is a fast, simple and clean video downloader built with Go. 

Annie helps users to download videos *from* supported websites, such as Youtube. With Annie, downloading videos and playlists has never been so easy!

Reply:
Considering that the title is Annie, I think this should be fine:

```
👾 A fast, simple and clean video downloader built with Go.

Annie helps users to download videos from supported websites, such as Youtube. With Annie, downloading videos and playlists has never been so easy!
```

`from` looks better 😄